jdbc-v2/src/main/java/com/clickhouse/data/Tuple.java

```diff
@@ -0,0 +1,35 @@
+package com.clickhouse.data;
+
+public class Tuple {
+    private final Object[] values;
+
+    public Tuple(Object... values) {
+        this.values = values;
+    }
+
+    public Object[] getValues() {
+        return values;
+    }
+
+    public Object getValue(int index) {
```

Comment:
I would add one more `getValue(column, targetType)` to be consistent with JDBC result set API